Kernel/FileSystem/FATFS/FileSystem.cpp

```diff
@@ -78,6 +78,18 @@ ErrorOr<void> FATFS::initialize_while_locked()
     root_entry.attributes = FATAttributes::Directory;
     m_root_inode = TRY(FATInode::create(*this, root_entry, { 0, 1 }));
 
+    m_fs_info = TRY(KBuffer::try_create_with_size("FATFS: FSInfo struct"sv, m_device_block_size));
+    auto fs_info_buffer = UserOrKernelBuffer::for_kernel_buffer(m_fs_info->data());
```

Comment:
Not a big deal but we could probably get away with inlining the structure in the FATFS class rather than using a heap buffer for it, since it's known to be 512 bytes and that's not that much.